dockerfiles/Dockerfile.apisix-runtime.rpm

```diff
@@ -30,7 +30,13 @@ ENV runtime_version=${RUNTIME_VERSION}
 
 COPY ${CODE_PATH} ./
 
-RUN mv ./utils/build-common.sh ./utils/determine-dist.sh ./ \
+RUN yum install -y curl \
```

Comment:
In the aarch64 platform architecture of the Centos7 platform, the wasmtime built by higher versions of Rust will fail.
be related to: https://github.com/bytecodealliance/wasmtime/issues/4848
 
 
![image](https://github.com/api7/apisix-build-tools/assets/79972061/6738fb87-d825-4f6d-b91a-94abc41f7321)

ci faield: https://github.com/api7/apisix-build-tools/actions/runs/7709143843/job/21010404999